packages/client/hmi-client/src/workflow/ops/document/tera-document-operation-drilldown.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+	<tera-drilldown :title="node.displayName" @on-close-clicked="emit('close')">
+		<div>
+			<tera-drilldown-section :is-loading="fetchingPDF">
+				<tera-pdf-embed v-if="pdfLink" :pdf-link="pdfLink" :title="document?.name || ''" />
+				<!-- <tera-text-editor v-else-if="view === DocumentView.TXT" :initial-text="docText" /> -->
```

Comment:
```suggestion
```

Reply:
Ive added a little functionality in here, for documents that aren't PDFs